keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -263,20 +289,24 @@ proposed resize operation for a given resource.  Any time the
 `Pod.Status.ContainerStatuses[i].Resources` field, this new field explains why.
 
 This field can be set to one of the following values:
-* Proposed - the proposed resize (in Spec...Resources) has not been accepted or
-  rejected yet.
-* InProgress - the proposed resize has been accepted and is being actuated.
-* Deferred - the proposed resize is feasible in theory (it fits on this node)
+* `Proposed` - the proposed resize (in Spec...Resources) has not been accepted or
+  rejected yet. `resources != allocatedResources`
+* `InProgress` - the proposed resize has been accepted and is being actuated. A new resize request
+  will reset the status to `Proposed`.
+  `resources == allocatedResources && allocatedResources != status.resources`
+* `Deferred` - the proposed resize is feasible in theory (it fits on this node)
   but is not possible right now; it will be re-evaluated.
-* Infeasible - the proposed resize is not feasible and is rejected; it will not
+  `resources != allocatedResources`
+* `Infeasible` - the proposed resize is not feasible and is rejected; it will not
   be re-evaluated.
+  `resources != allocatedResources`
 * (no value) - there is no proposed resize
```

Comment:
Minor comment Should we use same definitions used also in the code ?

```
const (
	// Pod resources resize has been requested and will be evaluated by node.
	PodResizeStatusProposed PodResizeStatus = "Proposed"
	// Pod resources resize has been accepted by node and is being actuated.
	PodResizeStatusInProgress PodResizeStatus = "InProgress"
	// Node cannot resize the pod at this time and will keep retrying.
	PodResizeStatusDeferred PodResizeStatus = "Deferred"
	// Requested pod resize is not feasible and will not be re-evaluated.
	PodResizeStatusInfeasible PodResizeStatus = "Infeasible"
)
```

Reply:
Should we add a new PodResizeStatus to cater when Pod resources resize has been accepted by node and has been actuated ( Completed ) ? when resources == allocatedResources == status.resources ( when allocatedResources is not empty ) . It was discussed in the past https://github.com/kubernetes/kubernetes/issues/123441#issuecomment-1978183702

Reply:
I think we should trigger an event in this case, but I'm not sure we need to keep around a completed status.